imod_coupler/__main__.py

```diff
@@ -34,17 +34,16 @@ def run_coupler(config_path: Path) -> None:
     with open(config_path, "rb") as f:
         config_dict = tomllib.load(f)
 
-    # Validation expects current working directory at config file level
-    os.chdir(config_path.parent)
-    base_config = BaseConfig(**config_dict)
+    config_dir = config_path.parent
+    base_config = BaseConfig(config_dir=config_dir, **config_dict)
```

Comment:
Could you repeat the comment here that the BaseConfig changes dir to ``config_dir`` at initialization here as well?